crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -42,24 +48,53 @@ pub struct StandardMaterial {
     /// it just adds a value to the color seen on screen.
     pub emissive: Color,
 
+    /// The [`emissive`] as a texture.
+    ///
+    /// This color is multiplied by [`emissive`] to get the final emitted color.
+    /// Meaning that you should set [`emissive`] to [`Color::WHITE`]
+    /// if you want to use an emissive texture.
+    ///
+    /// [`emissive`]: StandardMaterial::emissive
     #[texture(3)]
     #[sampler(4)]
     pub emissive_texture: Option<Handle<Image>>,
-    /// Linear perceptual roughness, clamped to [0.089, 1.0] in the shader
-    /// Defaults to minimum of 0.089
+
+    /// Linear perceptual roughness, clamped to `[0.089, 1.0]` in the shader.
+    ///
+    /// Defaults to minimum of `0.089`.
+    ///
+    /// Low values result in a "glossy" material with specular highlights,
+    /// while values close to `1` result in rough materials.
+    ///
     /// If used together with a roughness/metallic texture, this is factored into the final base
-    /// color as `roughness * roughness_texture_value`
+    /// color as `roughness * roughness_texture_value`.
     pub perceptual_roughness: f32,
-    /// From [0.0, 1.0], dielectric to pure metallic
+
+    /// How "metallic" the material appears, within `[0.0, 1.0]`,
+    /// going from dielectric to pure metallic.
+    ///
+    /// Defaults to `0.01`.
+    ///
+    /// The closer to `1` the value, the more the material will
+    /// reflect light like a metal such as steel or gold.
+    ///
     /// If used together with a roughness/metallic texture, this is factored into the final base
-    /// color as `metallic * metallic_texture_value`
+    /// color as `metallic * metallic_texture_value`.
     pub metallic: f32,
 
+    /// A texture representing both [`metallic`] and [`roughness`].
+    ///
+    /// The blue channel is the [`metallic`] and green is [`roughness`].
```

Comment:
Can we add a clarification that this is modulated by the scalar values? Because again this phrasing seems to imply this overrides them.

Reply:
I don't agree that "represents" implies "overrides." And the context should give enough information, I can't really write every sentence as if completely out of context. And it's very awkward to phrase "is actually multiplied with their respective scalar value" succinctly. You are welcome if you can come up with something though.

Though this reminds me there is a pitfall similar to `emissive` with the `metallic_roughness_texture` I already encountered twice and need to explain here.

Reply:
> I can't write every sentence as if completely out of context.

True. But equally true is the fact most users land on this documentation from a link, and will not have read the docs for other fields. So although we shouldn't repeat everything, there's an effort to make to 1) write the minimal amount of critical imformation, and 2) link to other sections of the docs that can help them get further details. As is, again, this documentation tells me that this texture is the unique source of truth of metallic and roughness, because "a texture representing a scalar fields" is quite unclear to me.

I'd suggest:

```
/// Metallic and roughness maps, stored as a single texture.
///
/// The blue channel contains metallic values, and the green channel roughness values.
/// Other channels are unused.
///
/// Those values are multipled by the scalar ones of the material; see [`metallic`] and
/// [`roughness`] for details.
///
/// [`metallic`]: StandardMaterial::metallic
/// [`roughness`]: StandardMaterial::perceptual_roughness
```

And that way the user knows they need to refer to those links to understand better how the values are mixed together.

Reply:
This is a really good doc string. Thanks for the help!